### ISSUE TITLE
403 error for org listing API call when switching to suborg 

### ISSUE THREAD

**Issue by AnuradhaSK** [wso2/product-is#20174]
**Describe the issue:**

- Build the latest IS pack using the master branch 
- Login to console 
- Create a sub org
- Switch to the sub org
- The following error message will be prompted
```
Error while getting the organization list
Operation is not permitted. You do not have permissions to make this request.
```

The failing request is 
```
https://localhost:9443/t/carbon.super/api/server/v1/organizations?after=&before=&filter=&limit=10&recursive=false

```
It invokes for super tenant path

https://github.com/wso2/product-is/assets/25483865/e38b5862-69e9-4bac-ac67-9c776a6050ff

```
        <!-- Identity Portal Versions -->
        <identity.apps.console.version>2.14.14</identity.apps.console.version>
        <identity.apps.myaccount.version>2.5.1</identity.apps.myaccount.version>
        <identity.apps.core.version>2.1.52</identity.apps.core.version>
        <identity.apps.tests.version>1.6.378</identity.apps.tests.version>
```

**Comment by brionmario** [comment]
There was a regression due to a change that went in `2.14.14` and was fixed in `2.14.15`.

https://github.com/wso2/identity-apps/releases/tag/%40wso2is%2Fconsole%402.14.15

**Comment by AnuradhaSK** [comment]
> There was a regression due to a change that went in `2.14.14` and was fixed in `2.14.15`.
> 
> https://github.com/wso2/identity-apps/releases/tag/%40wso2is%2Fconsole%402.14.15

Closing as it is not reproducible with console version 2.14.15
